Nodejs/Product/TestAdapter/TestFrameworks/mocha/mocha.js

```diff
@@ -205,7 +206,7 @@ function getMochaOptions(projectFolder) {
     }
 
     // set timeout to 10 minutes, because the default of 2 sec is too short for debugging scenarios
-    if (typeof v8debug === 'object') {
+    if (inspector.url()) {
```

Comment:
Thanks for the fix.

I noticed that the `inspector` module is marked as `Stability: 1 - Experimental` and not recommended for production. Instead, I think is a good idea to follow TypeScript's implementation.

Ignore the VS Code variables. Here's the implementation: https://github.com/microsoft/TypeScript/blob/598e9b2e88153399bab4902f986910226b0a8496/src/compiler/sys.ts#L1252